src/components/views/ProductList/SingleProduct/SingleProduct.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import styles from './SingleProduct.module.scss';
+import PropTypes from 'prop-types';
+import ProductRating from '../../../common/ProductRating/ProductRating';
+import Button from '../../../common/Button/Button';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faExchangeAlt,
+  faSearch,
+  faShoppingBasket,
+} from '@fortawesome/free-solid-svg-icons';
+import { faStar as faHeart } from '@fortawesome/free-regular-svg-icons';
+
+const SingleProduct = ({
```

Comment:
MOże nazwij to `ProductListItem` żeby było wiadomo że to na liście produktów jest wyświetlanie